From 378560050a8f93786c590cc99a55461666205b61 Mon Sep 17 00:00:00 2001
From: Xin Long <lucien.xin@gmail.com>
Date: Fri, 24 Aug 2018 01:13:32 +0800
Subject: [PATCH] build: fix netinet/sctp.h not to be installed

After libcnetinet_HEADERS was set to sctp.h.in, netinet/sctp.h can
no longer be installed into ${includedir}.

Since "AC_CONFIG_HEADERS([src/include/netinet/sctp.h])" is already
added into configure.ac, there's no need to generate sctp.h by
automake.

So we simply set libcnetinet_HEADERS back to sctp.h.

Fixes: 9607dd85e70a ("netinet/sctp.h: dynamically build based on system setup")
Signed-off-by: Xin Long <lucien.xin@gmail.com>
Signed-off-by: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>
---
 src/include/netinet/Makefile.am | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

--- a/src/include/netinet/Makefile.am
+++ b/src/include/netinet/Makefile.am
@@ -11,5 +11,4 @@ libcnetinetdir = $(includedir)/netinet
 # API.
 include_HEADERS =
 
-libcnetinet_HEADERS = sctp.h.in
-BUILT_SOURCES = sctp.h
+libcnetinet_HEADERS = sctp.h
